2d/numericalTanks/linearWaves/tank2D.py

```diff
@@ -200,7 +193,7 @@
     rd_shockCapturingFactor  = 0.5
     rd_lag_shockCapturing = False
     epsFact_density    = 3.0
-    epsFact_viscosity  = epsFact_curvature  = epsFact_vof = epsFact_consrv_heaviside = epsFact_consrv_dirac = epsFact_density
+    epsFact_viscosity  = epsFact_curvature  = epsFact_vof = ecH = epsFact_consrv_dirac = epsFact_density
```

Comment:
I know there were discussions about what to do with WaveTools and other things needing a variable named `ecH`.  What are the plans in that regard?  If we want to stick (for style or convenience) with `ecH`, how should this naming convention work with the other variables here (which similarly pop up in a lot of the air-water-vv set)?